.changeset/hot-cheetahs-press.md

```diff
@@ -0,0 +1,11 @@
+---
+'@sumup/circuit-ui': minor
```

Comment:
Right, it's a breaking change indeed... How about I add an `orientation` prop (vertical by default)? This would be backward compatible. This would clash with `inlineMobile` but, actually, might be better to get rid of it altogether. Wdyt?

Reply:
So, I got rid of `inlineMobile` prop and added `orientation` which is `vertical` by default, which is matching the current behavior. `stretch` is now `true` for the same also to match how it's rendered right now. With this, it should be a non-breaking change.

Reply:
You're, it's a mojor, as agreed previous, I just forgot to update this file 🙂

About the defaults, I think it makes sense to change them now to cover more common scenarious. Will do.